StoreApp.BLL/Services/OrderService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using AutoMapper;
+using StoreApp.BLL.DTO;
+using StoreApp.BLL.Interfaces;
+using StoreApp.DAL.Entities;
+using StoreApp.DAL.Intefaces;
+using System.Collections.Generic;
+
+namespace StoreApp.BLL.Services
+{
+    class OrderService : IOrderService
+    {
+        IUnitOfWork DataBase;
```

Comment:
1. The same as the previous comment
2. The previous example was a property but now it's a field. The best practice is to use the same approaches.

Reply:
fixed